sci-physics/geant-vmc/geant-vmc-4.3.6.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
This is the kind of change that really desires a revbump (`ebump -c` is your friend).

Reply:
Thanks, also for the hint to `ebump` which I did not know about yet!